genesis/Cargo.toml

```diff
@@ -12,15 +12,17 @@ workspace = true
 
 [dependencies]
 iroha_crypto = { workspace = true }
-iroha_data_model = { workspace = true, features = ["http"] }
+iroha_data_model = { workspace = true, features = ["http", "transparent_api"] }
```

Comment:
Is it OK to use `transparent_api` here? @mversic @Erigara

Reply:
As it was necessary to somehow setup a creation time I decided to make it public in the module with this feature. Maybe I'm missing some feature that would enable me to do just that without `transparent_api`.

Reply:
there is no need to enable `transparent_api` here. You can use `TransactionPayload::new_with_time`. This feature should only be enabled by `iroha_core`

@0x009922 I was actually thinking to rename this feature to `_transparent_api` to give people the idea it's an internal feature and dissuade them from using it. What do you think about this?

Reply:
@mversic, it would make sense, I think. And if it's really only for `iroha_core`, naming it `_transparent_api_for_iroha_core` would be clearer. I myself haven't understood the scope of it until now.